x-pack/plugins/security_solution/cypress/e2e/urls/not_found.cy.ts

```diff
@@ -30,7 +30,8 @@ describe('Display not found page', () => {
     visit(TIMELINES_URL);
   });
 
-  it('navigates to the alerts page with incorrect link', () => {
+  // TODO: We need to determine what we want the behavior to be here
```

Comment:
Can we open a ticket to track this skipped test?

Reply:
https://github.com/elastic/kibana/issues/143705

Reply:
NIT: Link the ticket in the comment.